src/components/header.tsx

```diff
@@ -11,13 +11,18 @@ const Header = ({ title }) => {
   return (
     <HeaderContainer>
       <HeaderBar>
-        <HeaderBrand>{title}</HeaderBrand>
-        {/* TODO: Figure out grid styling for login button */}
-        {/* <LeftNavItems>
-          <span>
-            <FontAwesomeIcon icon="sign-in-alt" /> Login to get started
-          </span>
-        </LeftNavItems> */}
+        <Link href="/">
+          <a>
+            <HeaderBrand>{title}</HeaderBrand>
+          </a>
+        </Link>
+        <LeftNavItems>
+          <Link href="/workspaces">
```

Comment:
Can we use a constant for URL so we can reuse the same source?

Reply:
Sure. Sounds good